src/main/webapp/maps/Place.js

```diff
@@ -158,7 +161,14 @@ class Place {
   }
 
   getInfoWindowContent() {
-    const content = '<h3>' + this._name + '</h3>';
+    let content = '<h3>' + this._name + '</h3>';
+    if (this._mapsPlace != null) {
+        content += '<h4> Place: ' + this._mapsPlace.name + '</h4>';
+    }
+    // ToUrlValue is favored instead of toString because it has a parameter
+    // to set the precision.
+    content+=
+        '<p>' + 'Coordinates: (' + this._position.toUrlValue(3) + ')' +'</p>';
```

Comment:
Can we use string interpolation here instead of concatenation? (Not sure if Javascript supports this, but I think it might)

Reply:
I used template literals.